public/locales/zh.json

```diff
@@ -184,6 +184,7 @@
   "withdrawPercentage": "提取的百分比",
   "withdrawSynth": null,
   "withdrewNToken": null,
+  "wrongNetworkContent": null,
```

Comment:
While we have added `null` translations to `zh.json` for other strings, I learned recently that this is not the correct approach. It is better *not* to make a null entry so that it defaults to English. 
I will make a PR to remove other null translations

Reply:
Oh, got it.